external/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ set(EXTERNAL_INCLUDES "")
 # fmtlib: https://github.com/chemfiles/fmt
 # ==========
 external_library(fmt)
-list(APPEND EXTERNAL_INCLUDES ${CMAKE_CURRENT_BINARY_DIR}/fmt)
+list(APPEND EXTERNAL_INCLUDES ${CMAKE_CURRENT_BINARY_DIR}/fmt/include/)
```

Comment:
It looks like you got #160 intermixed in this PR. Could you rebase on top of master?